TacTok/count_names.py

```diff
@@ -51,42 +52,65 @@ def count(filename, proof_data):
 
     # count occurrences within a goal
     def count_in_goal(node):
-        if is_name(node):
+        if SyntaxConfig.is_path(node):
+            # paths
+            if syn_conf.include_paths:
+                for c in node.children:
+                    ident = c.children[0].data
+                    incr_ident(ident, paths)
+                    incr_ident(ident, merged)
+            node.children = [] # don't treat these like global definitions
+        elif syn_conf.include_defs and SyntaxConfig.is_ident(node):
```

Comment:
is_path is an additional case for paths, split from idents. is_local is what refers to constructor_var; that should never show up in a definition or path, those are separate, as they were in is_name before

Reply:
Also, count_names now constructs all of these vocabularies at once for whatever options are enabled, rather than forcing you to run it several times. is_name before had that if/else because it was using the old options to decide which identifiers to get, and could only get one kind at a time, but that's silly.